src/modules/questionnaires/questionnaire_entity.ts

```diff
@@ -69,96 +82,96 @@ export namespace QuestionnaireEntity {
   }
 
   export interface Properties {
-    pemohon: Pemohon
-    fasilitas_desa: FasilitasDesa
-    literasi_digital: LiterasiDigital
-    tentang_bumdes: TentangBumdes
-    potensi_desa: PotensiDesa
+    applicant: Applicant
+    facility: Facility
+    literacy: Literacy
+    bumdes: PropertiesBumdes
+    potential: Potential
   }
 
-  export interface FasilitasDesa {
-    akses_kendaraan: AksesKendaraan
-    suplai_listrik: SuplaiListrik
-    jaringan_telepon: JaringanTelepon
-    jaringan_internet: JaringanInternet
+  export interface Applicant {
+    name: string
+    position: string
+    phone_number: string
+    email: string
   }
 
-  export interface AksesKendaraan {
-    data: string[]
-    photo: File
+  export interface PropertiesBumdes {
+    social_media: SocialMedia
+    bumdes: BumdesBumdes
+    commodity: Commodity
+    ecommerce: Ecommerce
+    logistics: string
   }
 
-  export interface File {
-    path: null | string
-    original_name: null | string
-    source: null | string
+  export interface BumdesBumdes {
+    data: string
```

Comment:
mang ini apa emang bumdes nya si bumdes alias BumdesBumdes apa Bumdes aja is enough ya 😢 😄

Reply:
typo sepertinya .. wkwkwk

Reply:
sudah di update yah [a545ca8](https://github.com/jabardigitalservice/dedi-core-service/pull/232/commits/a545ca8edfe2f0ce4bf2bc2de6431d554153dc51)